sdk/formrecognizer/ai-form-recognizer/src/documentAnalysisClient.ts

```diff
@@ -608,63 +627,78 @@ export class DocumentAnalysisClient {
     const toInit =
       // If the user gave us a stored token, we'll poll it again
       resumeFrom !== undefined
-        ? async () => {
-            const { operationLocation, modelId } = JSON.parse(resumeFrom) as {
-              operationLocation: string;
-              modelId: string;
-            };
+        ? async () =>
+            this._tracing.withSpan(
+              "DocumentAnalysisClient.createAnalysisPoller-resume",
```

Comment:
I wonder if `resume` and `start` postfix should instead be a span attribute - @lmolkova what do you think? 

My reasoning here is to follow the low cardinality of span names recommendation - "The span name SHOULD be the most general string that identifies a (statistically) interesting class of Spans, rather than individual Span instances while still being human-readable. That is, “get_user” is a reasonable name, while “get_user/314159”, where “314159” is a user ID, is not a good name due to its high cardinality. Generality SHOULD be prioritized over human-readability." 

From https://opentelemetry.io/docs/reference/specification/trace/api/#span

Reply:
Thanks, @maorleger!  having `resume` and `start` in the span name is actually fine - it's still low-cardinality. E.g. for HTTP we include the verb into span name and it sounds similar. I'd assume users would benefit from having `resume` and `start` on the trace visualizations and span name is commonly used there.

Having it as an attribute makes sense too, but attribute names are somewhat hard to agree upon and ""I propose to keep verb in the span name**, but I'd rather change it to `start DocumentAnalysisClient.createAnalysisPoller`,  `resume DocumentAnalysisClient.createAnalysisPoller`. Similarly to [http](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/http.md) (`GET /path`) and [database](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md) (`SELECT ShopDb.orders`) Otel conventions.

Reply:
I don't know if putting start at the beginning would be right here. It's not really a generic method in the same way that a GET request is generic for HTTP, it's really a subroutine of `createAnalysisPoller`.